Bottom: 8e77a6acf6ddb289241e3275b2ee40d8e85c5f89
Top:    ab3a0ec563df017c98369f226a299675419b0439
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-24 23:14:19 +0100

Refresh of hid-gtk-simplify-viewport-setu

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 02129e6..bab13ce 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1147,9 +1147,7 @@ PCBChanged (int argc, char **argv, Coord x, Coord y)
     return 0;
   RouteStylesChanged (0, NULL, 0, 0);
   ghid_port_ranges_scale ();
-  ghid_port_ranges_pan (0, 0, FALSE);
   ghid_zoom_view_fit ();
-  ghid_port_ranges_changed ();
   ghid_sync_with_new_layout ();
   return 0;
 }
